bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeHandler.java

```diff
@@ -276,7 +279,12 @@ private void refresh() {
             updateGridPowerValues(getSenecValue(response.grid.currentGridValue));
 
             updateStatus(ThingStatus.ONLINE);
-        } catch (IOException | InterruptedException | TimeoutException | ExecutionException e) {
+        } catch (JsonSyntaxException | IOException | InterruptedException | TimeoutException | ExecutionException e) {
```

Comment:
This line is the actual fix. The other lines are again a huge help in case something is wrong with the response.